lib/govuk_ab_testing/rspec_capybara_helpers.rb

```diff
@@ -0,0 +1,33 @@
+module GovukAbTesting
+  module RspecCapybaraHelpers
+    def with_variant(args)
+      unless defined?(page)
```

Comment:
I'm not sure if different versions of capybara would have a different class for `page`. I could check if it is a `Capybara::Session`, but not sure if we would gain much by doing it.